packages/core/utils/src/progress-message.js

```diff
@@ -6,7 +6,7 @@ import path from 'path';
 export function getProgressMessage(event: BuildProgressEvent): ?string {
   switch (event.phase) {
     case 'transforming':
-      return `Building ${path.basename(event.filePath)}...`;
+      return `Transforming ${path.basename(event.filePath)}...`;
```

Comment:
@devongovett This was always a bit of misleading, maybe a remnant from Parcel 1? Or should we leave it as is?

Reply:
Let's leave it as is for now so that we can merge the rest. 

```suggestion
      return `Building ${path.basename(event.filePath)}...`;
```